subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm.go

```diff
@@ -44,10 +44,20 @@ type HivedAlgorithm struct {
 	vcSchedulers map[api.VirtualClusterName]intraVCScheduler
 	// scheduler for opportunistic pods
 	opportunisticSchedulers map[CellChain]*topologyAwareScheduler
-	// ChainCellLists of physical cells of each cell chain
+	// ChainCellLists of physical cells of each cell chain (including the children of the free cells)
 	fullCellList map[CellChain]ChainCellList
 	// ChainCellLists of free physical cells of each cell chain (used in buddy alloc)
 	freeCellList map[CellChain]ChainCellList
+	// remaining quota of each VC of each cell type
+	vcRemainingQuotas map[api.VirtualClusterName]map[CellChain]map[CellLevel]int32
+	// total remaining quota of each cell type
+	totalRemainingQuotas map[CellChain]map[CellLevel]int32
```

Comment:
What is the quota unit? Better to rename as more concert things, such as vcRemainingCells, vcRemainingXXXXs

Reply:
The unit is cell, or more precisely, preassigned cells.
How about vcRemainingPreassignedCells?

Reply:
or vcFreePreassignedCells